pkg/controller/pxc/users.go

```diff
@@ -160,6 +160,12 @@ func (r *ReconcilePerconaXtraDBCluster) reconcileUsers(cr *api.PerconaXtraDBClus
 		return nil, errors.Wrap(err, "manage sys users")
 	}
 
+	// we should update replication password in internal secret only after updating it for replication channels
+	for k, v := range sysUsersSecretObj.Data {
+		if k != "replication" {
+			internalSysSecretObj.Data[k] = v
+		}
+	}
 	internalSysSecretObj.Data = sysUsersSecretObj.Data
```

Comment:
does it make sense ?

Reply:
@hors fixed https://github.com/percona/percona-xtradb-cluster-operator/pull/1164/commits/0579ff5ccfc1182357f953a3795f646436742c9f